app/workers/activitypub/forward_distribution_worker.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class ActivityPub::ForwardDistributionWorker < ActivityPub::DistributionWorker
+  include Sidekiq::Worker
+
+  sidekiq_options queue: 'push'
+
+  def perform(conversation_id, json)
+    conversation = Conversation.find(conversation_id)
+
+    @status  = conversation.parent_status
```

Comment:
What if this isn't available because that status was deleted? Need to cancel then.